### PR TITLE
F.Fernando

### PR DESCRIPTION
**Pull request by FJF616**
Still more to be done

### CODE REVIEW

File: project/ls8/cpu.js [hunk 1]
```diff
@@ -3,14 +3,43 @@
...
+const AND = 0b10110011;  // AND R R
+const NOP = 0b00000000;  // NOP
+const NOT = 0b01110000;  // NOT R
+const XOR = 0b10110010;  // XOR R R
+const OR = 0b10110001;   // OR R R
+const CALL = 0b01001000; // CALL R
+const CMP  = 0b10100000; // CMP R R
+const DEC  = 0b01111001; // DEC R
+const DIV  = 0b10101011; // DIV R R
+const HLT  = 0b00000001; // Halt CPU
```
Comment (BlackBelt2817) [inline project/ls8/cpu.js]: HLT is already declared

File: project/ls8/cpu.js [hunk 2]
```diff
@@ -3,14 +3,43 @@
...
+const JEQ  = 0b01010001; // JEQ R
+const JGT  = 0b01010100; // JGT R
+const JLT  = 0b01010011; // JLT R
+const JMP  = 0b01010000; // JMP R
+const JNE  = 0b01010010; // JNE R
+const LD   = 0b10011000; // Load R,R
+const MOD  = 0b10101100; // MOD R R
+const POP  = 0b01001100; // Pop R
+const PRA  = 0b01000010; // Print alpha char
+const PRN  = 0b01000011; // Print numeric register
```
Comment (BlackBelt2817) [inline project/ls8/cpu.js]: PRN is already declared

File: project/ls8/cpu.js [hunk 3]
```diff
@@ -134,14 +310,98 @@ class CPU {
...
+    }
+    /**
+       * JNE R
+       */
+    JNE(regNum) {
+        // if equal flag is false, execute at specified register
+        if (!this.getFlag(FLAG_EQ)) {
+            return this.reg[regNum]; 
+    }
+
```
Comment (BlackBelt2817) [inline project/ls8/cpu.js]: You'll want to add one more closing curly bracket for JNE(regNum) here.

File: project/ls8/cpu.js (hunk 2 above)
Comment (BlackBelt2817) [inline project/ls8/cpu.js]: PRN is already declared.

File: project/ls8/cpu.js (hunk 1 above)
Comment (BlackBelt2817) [inline project/ls8/cpu.js]: HLT is already declared.

**Comment by FJF616** [comment]
Thanks!

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Pull request closed without merging**